addons/website_blog/views/website_blog_templates.xml

```diff
@@ -579,9 +579,9 @@
 
 <!-- Chatter templates -->
 <template id="blog_post_template_new_post">
-    <p>A new post <t t-esc="post.name" /> has been published on the <t t-esc="object.name" /> blog. Click here to access the blog :</p>
+    <p>A new post <t t-esc="post.name" /> has been published on the <t t-esc="object.name" /> blog. Click here to access it :</p>
```

Comment:
please don't modify this part, changing the content would break the translations